site/content/docs/Repository Server/_index.md

```diff
@@ -54,6 +54,7 @@ kopia server start \
   --address 0.0.0.0:51515 \
   --server-control-username control \
   --server-control-password PASSWORD_HERE
+  --password REPO_PASSWORD
```

Comment:
The preferred, recommended way to specify the passwords is via an environment variables, instead of via a CLI flags. Please update the PR.

```shell
KOPIA_PASSWORD="<password-for-the-repository>" \
KOPIA_SERVER_CONTROL_PASSWORD="<server-control-password>" \
  kopia server start \
    --tls-generate-cert \
    --tls-cert-file ~/my.cert \
    --tls-key-file ~/my.key \
    --address 0.0.0.0:51515 \
    --server-control-username control
```